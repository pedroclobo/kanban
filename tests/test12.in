t 12 análise de requisitos
t 8 desenho
t 34 realização
t 24 teste
t 17 instalação
l 5 4 3 2 1
q
