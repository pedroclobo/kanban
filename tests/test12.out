task 1
task 2
task 3
task 4
task 5
5 TO DO #17 instalação
4 TO DO #24 teste
3 TO DO #34 realização
2 TO DO #8 desenho
1 TO DO #12 análise de requisitos
